cmd/xmtpd-cli/commands/funds.go

```diff
@@ -330,7 +330,7 @@ func mintHandler(to common.Address, amountStr string, raw bool) error {
 
 	logger.Info("successfully minted mock underlying fee token",
 		zap.String("to", to.Hex()),
-		zap.String("amountRaw", amount.String()),
+		zap.String("amount_raw", amount.String()),
```

Comment:
I guess neither of these names is great. The user passes in `--amount` so we could use that. Or if we wanted to remind them its raw `amount (raw)` maybe?